examples/hacker_news_pwa/web/index.html

```diff
@@ -9,6 +9,7 @@
   <title>Hacker News - AngularDart PWA</title>
   <script defer src="main.dart.js"></script>
   <link rel="manifest" href="manifest.json">
+  <link rel="preload" href="https://node-hnapi.herokuapp.com/news?page=1" as=fetch crossorigin >
```

Comment:
This is interesting. I'm concerned a bit its a "waste" if not navigating to that page, but I think for a demo (which this is) it's probably the right call. I'll move this into my PR.